clab/docker.go

```diff
@@ -177,9 +177,12 @@ func (c *CLab) CreateContainer(ctx context.Context, node *Node) (err error) {
 		NetworkMode:  container.NetworkMode(c.Config.Mgmt.Network),
 	}
 
-	containerNetworkingConfig := new(network.NetworkingConfig)
+	containerNetworkingConfig := &network.NetworkingConfig{}
 
-	if !node.NetworkModeHost {
+	switch node.NetworkMode {
+	case "host":
+		containerHostConfig.NetworkMode = container.NetworkMode("host")
+	case "default":
```

Comment:
I'm confused here. Do you somewhere set NetworkMode to "default"?
Or was this meant to be  `default:` instead of `case "default:"` ?

Reply:
sorry, it should read

```
	switch node.NetworkMode {
	case "host":
		containerHostConfig.NetworkMode = container.NetworkMode("host")
	default:
```